next-cmake/rocisa/setup.py

```diff
@@ -0,0 +1,66 @@
+import glob
+import os
+import shutil
+import sys
+import subprocess
+from pathlib import Path
+from setuptools import setup, Extension
+from setuptools.command.build_ext import build_ext
+
+ROCISA_ROOT= str(Path(__file__).parent.parent.parent / "tensilelite" / "rocisa")
+class CMakeBuild(build_ext):
+    def run(self):
+        try:
+            os.makedirs(self.build_temp, exist_ok=True)
+            os.makedirs(self.build_lib, exist_ok=True)
+            source_dir = os.path.abspath(os.path.dirname(__file__))
+            rocm_path = os.environ.get('ROCM_PATH', '/opt/rocm')
+            compilerpath = os.path.join(rocm_path, 'bin/amdclang++')
```

Comment:
PSA: `amdclang++` isn't a real thing. It is just a convenience symlink we provide in some builds.

(fine to leave for now)